packages/web3-eth-iban/src/iban.ts

```diff
@@ -22,8 +22,26 @@ import { InvalidAddressError } from 'web3-errors';
 import { IbanOptions } from './types.js';
 
 /**
- * Converts Ethereum addresses to IBAN or BBAN addresses and vice versa.
- */
+ * Converts Ethereum addresses to IBAN or BBAN addresses and vice versa.<br/><br/>
```

Comment:
Unfortnatly, this `<br/>` is not converted to a new line. Could you please remove it from each place it is used? 
```suggestion
 * Converts Ethereum addresses to IBAN or BBAN addresses and vice versa.
```
Thanks,